From: David Lutz <kpanic@hirnduenger.de>
Date: Wed, 5 Oct 2016 10:46:36 +0200
Subject: This patch adds support for the TP-Link TL-WR843N/ND to trunk.

diff --git a/target/linux/ar71xx/image/Makefile b/target/linux/ar71xx/image/Makefile
index cbd8a49..58393b3 100644
--- a/target/linux/ar71xx/image/Makefile
+++ b/target/linux/ar71xx/image/Makefile
@@ -625,13 +625,20 @@ define Device/tl-wr843nd-v1
     TPLINK_HWID := 0x08430001
 endef
 
+define Device/tl-wr843n-v2
+    $(Device/tplink-4mlzma)
+    BOARDNAME := TL-WR841N-v11
+    DEVICE_PROFILE := TLWR843
+    TPLINK_HWID := 0x08430002
+endef
+
 define Device/tl-wr847n-v8
     $(Device/tplink-4mlzma)
     BOARDNAME := TL-WR841N-v8
     DEVICE_PROFILE := TLWR841
     TPLINK_HWID := 0x08470008
 endef
-TARGET_DEVICES += tl-wr841n-v8 tl-wr841n-v9 tl-wr841n-v10 tl-wr841n-v11 tl-wr842n-v2 tl-wr842n-v3 tl-wr843nd-v1 tl-wr847n-v8
+TARGET_DEVICES += tl-wr841n-v8 tl-wr841n-v9 tl-wr841n-v10 tl-wr841n-v11 tl-wr842n-v2 tl-wr842n-v3 tl-wr843nd-v1 tl-wr843nd-v2 tl-wr847n-v8
 
 define Device/tl-wr941nd-v5
     $(Device/tplink-4mlzma)
